rust/routee-compass-core/src/model/road_network/edge_heading.rs

```diff
@@ -0,0 +1,68 @@
+use serde::Deserialize;
+
+#[derive(Copy, Clone, Deserialize)]
+pub struct EdgeHeading {
+    start_heading: i16,
+    end_heading: Option<i16>,
```

Comment:
future work we could make Headings a Unit type in RouteE Compass

Reply:
Yeah that's a good idea